### PR TITLE
Update FTW documentation

### LINKED ISSUE #2696: Number of Classes in FieldsOfTheWorld Dataset
### Description

I am iterating through the 3-class version of the FieldsOfTheWorldDataset and noticing that for the country of brazil and sometimes india the max label value is 3, so others return [0, 1, 2] but for brazil [1, 2, 3] are returned as mask labels. 

Do you perhaps have any insights @calebrob6 ?

### Steps to reproduce

```python

ds = FieldsOfTheWorld(root="FieldsOfTheWorld", target='3-class', countries=CC_BY_COUNTRIES)

dl = DataLoader(ds, batch_size=64, num_workers=8)

for batch in tqdm(dl):
    if batch["mask"].max() > 2:
        print("Max value in mask is greater than 2")
```

or check these two examples directly:

```python
path = "FieldsOfTheWorld/brazil/label_masks/semantic_3class/g3_0000011264-0000001536.tif"
path = "FieldsOfTheWorld/india/label_masks/semantic_3class/g1_1799.tif"

with rasterio.open(path) as src:
    mask = src.read(1)
    print(mask.max())
    print(np.unique(mask))

```

### Version

'0.7.0.dev0'

### PR DESCRIPTION
**Pull request by calebrob6**
Closes #2696

### CODE REVIEW

**Review by adamjstewart — COMMENTED** [review]
Should we update `docs/api/datasets/non_geo_datasets.csv` too?

**Review by adamjstewart — APPROVED** [review]
